mne/utils.py

```diff
@@ -752,6 +752,35 @@ def _check_mayavi_version(min_version='4.3.0'):
         raise RuntimeError("Need mayavi >= %s" % min_version)
 
 
+def _check_pyface_backend():
```

Comment:
`_check_pyface_backend()` would be used by the test decorate, and `_check_backend()` has a GUI element (the warning dialog), so that would involve mixing GUI and non-GUI stuff...